src/components/connect.tsx

```diff
@@ -34,7 +34,10 @@ import defaultMapStateToPropsFactories from '../connect/mapStateToProps'
 import defaultMergePropsFactories from '../connect/mergeProps'
 
 import { createSubscription, Subscription } from '../utils/Subscription'
-import { useIsomorphicLayoutEffect } from '../utils/useIsomorphicLayoutEffect'
+import {
+  useIsomorphicLayoutEffect,
+  canUseDOM,
```

Comment:
If I see correctly this is a leftover - this actually ain't used in this file atm.

Reply:
Sort of the other way around. I made this change speculating I might need to use this in `connect` somehow, but I haven't made any actual code changes to `connect` yet - that's why this is a WIP PR.